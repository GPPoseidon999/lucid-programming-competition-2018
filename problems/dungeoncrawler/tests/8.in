23
P
A A 2 B D
B B 0
C C 4 A M Q D
D D 1 V
E E 0
F F 3 H G E
G G 1 V
H H 1 J
I I 1 V
J J 0
K K 0
L L 1 J
M M 4 D K I J
N N 0
O O 0
P P 4 O V R T
Q Q 0
R R 4 J L C F
S S 3 N U V
T T 1 S
U U 0
V V 1 W
W W 0