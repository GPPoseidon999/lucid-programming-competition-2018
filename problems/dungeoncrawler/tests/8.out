O W V N U S T J L E G H F D Q B A I K M C R P